data/almaMmsId2rpbId.tsv

```diff
@@ -1,7 +1,7 @@
 almaMmsId	rpbId	hbzId	zdbId
-990055779300206441	RPB036t940093	HT007024140	919150-1
+990055779300206441	RPB929t993394	HT007024140	919150-1
```

Comment:
New workflow keeps the `almaMmsId` and `rpbId` and just added the `hbzId` and `zdbId` from lobid. By that I was able to prefent the overwriting  described on the left.

Compare line 2 from the old file:
https://github.com/hbz/lookup-tables/blob/c6f9125761c97167ebb80a9a84e6fe92de500766/data/almaMmsId2rpbId.tsv?plain=1#L2

Number of lines stays the same.